tools/sidekick/config.json

```diff
@@ -76,6 +76,14 @@
       "paletteRect": "top: auto; bottom: 20px; left: 20px; height: 498px; width: 460px;",
       "includePaths": [ "**.docx**", "**.xlsx**" ]
     },
+    {
+      "containerId": "tools",
+      "title": "Send to CaaS",
+      "id": "sendtocaas",
+      "environments": ["dev","preview", "live", "prod"],
```

Comment:
Shouldn't the environments be only "live" and "prod"?

Reply:
Tbh I am not sure. I just copied the same configs that BACOM used.

Reply:
Having dev and preview shouldn't be an issue.